.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged
```

Comment:
huh is that a new way of handling husky? 🤔

I'm curious

Reply:
yep, husky changed the configuration completely in v6. here's their blog post about it:
https://blog.typicode.com/husky-git-hooks-javascript-config/